tests/conftest.py

```diff
@@ -235,6 +238,8 @@ def application_docker_image_configs() -> Mapping[str, Dict[str, Any]]:
         },
         "java": {
             "": {},
+            "hotspot-jdk-8": {},  # add for clarity when testing with multiple JDKs
```

Comment:
To make sure it doesn't break if the default image is changed, you can pass here `"JAVA_BASE_IMAGE": "openjdk:8u275"`

Reply:
but we don't specify version or hash in openjdk11, so I prefer either to pin them both to one version or leave them both "unpinned"

Reply:
Fair. okay